lib/teecket.rb

```diff
@@ -22,7 +21,5 @@ def self.search(params)
     malindo.get
 
     output = airasia.fares + mas.fares + malindo.fares + firefly.fares
```

Comment:
When it becomes the last line, we can just use airasia.fares + mas.fares + malindo.fares + firefly.fares (without the output).

It'd be great if you update this one so that I can merge this pull request.